North-America

```diff
@@ -1,2 +1,3 @@
 USA
 Canada
+Mexico
```

Comment:
Is Mexico a North American country? I think it's south american. Can we check the map before committing the code?

Reply:
checked the map. it is a north american country.